scientific/src/commonMain/kotlin/converter/kinematicViscosity/convertToTime.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ Copyright 2022 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.scientific.converter.kinematicViscosity
+
+// TODO Implement these after this is fixed https://github.com/splendo/kaluga/issues/453
```

Comment:
Since its not implemented, please remove this commented out part. Its not there for all other reciprocal units either.

Reply:
removed them